addon/components/item-picker/layer-picker/component.js

```diff
@@ -11,7 +11,15 @@
 
 import Component from '@ember/component';
 import layout from './template';
-
+import { inject as service } from '@ember/service';
 export default Component.extend({
-  layout
+  layout,
+  intl: service(),
+  /**
+   * Compute the i18n key for the geometry type and return that
+   */
+  geometryTypeKey: computed('layer.geometryType', function () {
+    let key = `$this.get('i18nScope')}.shared.geometryType.${this.get('layer.geometryType')}`;
```

Comment:
I think this is missing a `{`

Reply:
good catch... and very weird that this worked when I was testing it... but I think you are correct and I'll check/PR a fix

Reply:
very weird... obviously you commented on the changes... but that file is now (basically) empty. The geometryTypeKey stuff got moved into the layer-picker-row component, and it's correct there... not sure how your diff was out of sync... anyhow - master is fine